30412/Giurgiu Razvan - Florin/Assignment_1/alloc.c

```diff
@@ -0,0 +1,173 @@
+//
+// Created by TheRa on 2/26/2020.
+//
+
+#include "alloc.h"
+
+
+void startSll()
+{
+    first=(NodeT*)malloc(sizeof(NodeT));
```

Comment:
This is a bit redundant - if you are allocating space for it but then assigning NULL to the same variables, what is the point of allocating space in the first place?